rakelib/test.rake

```diff
@@ -8,15 +8,15 @@ namespace "test" do
   end
 
   task "core" do
-    run_rspec(Rake::FileList["spec/**/*_spec.rb"])
+    exit run_rspec(Rake::FileList["spec/**/*_spec.rb"])
```

Comment:
why not call exit in last statement of #run_spec, instead of having 3 exit calls?